### ISSUE TITLE
Option to turn off colors in console output

### ISSUE THREAD

**Issue by v-srinivasan-sfdc** [buildpacks/lifecycle#312]
While posting the build logs in emails or any other HTML based output, the logs looks like this.
```
�[33;1mWarning: �[0mNot restoring or caching layer data, no cache flag specified.
```
It will be great to have a `--no-color` option similar to `pack`

**Comment by ekcasey** [comment]
We were going to make the default color behavior depend on the existence of a `tty` (once `pack` actually runs the lifecycle containers with a `tty`) but an explicit flag is a good idea regardless.

https://github.com/buildpacks/pack/issues/493

**Comment by ekcasey** [comment]
Resolved by #328
